test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -68,16 +68,16 @@
 try:
     import psycopg2
     PSYCOPG = True
-except:
+except BaseException:
```

Comment:
Why the change from the blanket excepts to BaseException?  The reason you don't do blanket excepts is because it catches things like KeyboardInterrupt.  BaseException includes that.  This isn't the right PR to do this change.  When we get rid of the blanket excepts in pbs_testlib.py, we should go through and actually look what the try/except block is expects and catch that.

Reply:
@bhroam , removed the changes.